indico/htdocs/sass/modules/_event_management.scss

```diff
@@ -501,9 +501,38 @@ th.i-table {
         }
     }
 
-    .table-content {
+    thead {
+        display: block;
+        position: relative;
+        background-color: $light-gray;
+        border-bottom: 1px solid $gray;
+    }
+
+    tbody {
+        display: block;
         max-height: 50em;
-        overflow: scroll;
+        overflow: auto;
+    }
+
+    td:nth-child(1), th:nth-child(1) { min-width: 2.5em; }  // checkbox
```

Comment:
Better than having comments that will become obsolete, why not simply adding per-column classes? Otherwise if we at some point we decide to change the order of the columns we'll need to update this set of rules.